UsersApp/View Controllers/UsersViewController.swift

```diff
@@ -28,6 +23,15 @@ class UsersViewController: UIViewController {
         fetchUsers()
     }
     
+    private func setUpNavBar() {
+        let titleAtributes: [NSAttributedString.Key: Any] = [
+            .font: UIFont.systemFont(ofSize: 22)
+        ]
+
+        navigationController?.navigationBar.titleTextAttributes = titleAtributes
+        navigationItem.title = "Users"
+    }
+    
     private func fetchUsers() {
         let networkManager = NetworkManager()
         Task {
```

Comment:
since you added the error extension into networkManager, it would be nice to use it for capturing that thrown error in fetchUsers() as well.